extensions/amp-access/0.1/test/test-amp-access.js

```diff
@@ -1036,7 +1036,8 @@ describe('AccessService pingback', () => {
       expect(service.reportViewToServer_.callCount).to.equal(0);
       expect(triggerEventStub.callCount).to.equal(triggerStart);
       firstAuthorizationResolver();
-      return service.firstAuthorizationPromise_;
+      return Promise.all([service.reportViewPromise_,
+          service.reportViewPromise_]);
```

Comment:
Why wait on the same promise twice? Did you mean to use `firstAuthorizationPromise_`?